opentreemap/opentreemap/celery.py

```diff
@@ -16,3 +19,12 @@
 # pickle the object when using Windows.
 app.config_from_object('django.conf:settings')
 app.autodiscover_tasks(lambda: settings.INSTALLED_APPS)
+
+
+@task_failure.connect
+def handle_task_failure(**kw):
+    rollbar_settings = getattr(settings, 'ROLLBAR', {})
+    access_token = rollbar_settings.get('access_token')
+    if access_token:
+        rollbar.init(access_token, rollbar_settings['environment'])
```

Comment:
This was printed to the gunicorn standard out:

```WARNING:rollbar:Rollbar already initialized. Ignoring re-init.```

Looks like rollbar handles it properly, but is telling us not to do it. I am not sure if we should bother keeping our own flag. What do you think?